src/Microsoft.TestPlatform.ObjectModel/Navigation/PortableSymbolReader.cs

```diff
@@ -148,4 +153,37 @@ private void PopulateCacheForTypeAndMethodSymbols(string binaryPath)
             throw;
         }
     }
+
+    /// <summary>
+    /// Reads the pdb data from the dlls itself, either by loading the referenced pdb file, or by reading
+    /// embedded pdb from the dll itself.
+    /// </summary>
+    /// <param name="binaryPath"></param>
+    /// <returns></returns>
+    /// <exception cref="InvalidOperationException"></exception>
+    private static PortablePdbReader CreatePortablePdbReaderFromPEData(string binaryPath)
+    {
+        var peReader = new PEReader(new FileStream(binaryPath, FileMode.Open, FileAccess.Read));
```

Comment:
file stream `using` or `MetadataReaderProvider` needs the open handle?

Reply:
Fixed.